docs/usage.rst

```diff
@@ -1,34 +1,38 @@
 Using ``pyramid_nacl_session``
 ==============================
 
-Generating a Random Secret
---------------------------
+Installation
+------------
 
-First, you need to arrange to have a 32-byte random secret available:
+Install using setuptools, e.g. (within a virtualenv)::
 
-.. code-block:: python
+  $ pip install pyramid_nacl_session
```

Comment:
I think that installation of this package is seldom done by `pip install pyramid_nacl_session`, but usually by requiring it in `setup.py` or other methods. I think that we would better serve users by being vague about how to install this package ("Install pyramid_nacl_session via your preferred method.") or remove this section altogether.

However if you wish to preserve this section, then it should be:

```
Install using pip.

.. code-block:: bash

    pip install pyramid_nacl_session
```